mypy/server/astdiff.py

```diff
@@ -282,9 +292,9 @@ def visit_deleted_type(self, typ: DeletedType) -> SnapshotItem:
 
     def visit_instance(self, typ: Instance) -> SnapshotItem:
         return ('Instance',
-                typ.type.fullname(),
+                encode_optional_str(typ.type.fullname()),
```

Comment:
Doesn't this mean there is a bug elsewhere? Maybe we shouldn't mask it (especially if there is no repro)?

Reply:
I think that this is not a great place to detect the bug, since the traceback won't tell much.